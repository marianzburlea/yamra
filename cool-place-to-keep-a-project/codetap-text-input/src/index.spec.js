import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import Input from '.';

describe('Input type text', () => {
    test('renders correctly', () => {
        const tree = renderer.create(<Input />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
