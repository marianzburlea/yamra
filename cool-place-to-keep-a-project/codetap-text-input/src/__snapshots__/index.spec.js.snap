// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input type text renders correctly 1`] = `
.c0 {
  border: 1px solid #036;
}

<input
  className="c0"
/>
`;
